src/components/Patient/SampleDetails.tsx

```diff
@@ -372,7 +372,7 @@ export const SampleDetails = ({ id }: DetailRoute) => {
                 {t("doctors_name")}:{" "}
               </span>
               <span id="doctor_name">
-                {startCase(camelCase(sampleDetails.doctor_name))}
+                {capitalizeWords(cleanString(sampleDetails.doctor_name || ""))}
```

Comment:
Why are we performing cleanString here?
Also can't we just use `capitalize` class instead here?

Reply:
cleanString is one of two main functionality of startcase
we can skip capitalizeWords, (i asked upside) but there is no way we can use some tailwind class to behave like cleanString

Reply:
we need to use it because eariler we were using startCase of lodash here and if we dont , cypress tests are failing

Reply:
doctor_name is a user inputted data. let's not perform clean string on it. Cypress test could be wrong here. Modify the test.

cc: @nihal467

Reply:
const doctorName = "Dr John Doe";

this is the expected doctor name
file - cypress/e2e/sample_test_spec/SampleTestRequest.cy.ts

Reply:
can i just remove the checking of doctor_name then?
@rithviknishad @nihal467

Reply:
actually cant remove it, it performs actions in the cypress browser and the name is needed
![image](https://github.com/user-attachments/assets/a8be05fd-c3bd-40df-a0e9-887f7d4d8095)
![image](https://github.com/user-attachments/assets/12c95841-fa79-43f2-a06b-180e3ed83eb0)

Reply:
Whatever is the user input for `doctor_name` field, should be the one that's shown in the details page as is. So modify the cypress test accordingly.